games-action/supertuxkart/supertuxkart-1.2.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="3D kart racing game"
+DESCRIPTION="SuperTuxKart is a free 3D kart racing game. It aims towards fun for 
+players of all skill levels, with item boxes giving random items, nitro, drifting and 
+more. Realism is not the focus.
+
+STK offers an online multiplayer mode, a local multiplayer mode, and a single player \
+mode against AIs with both custom races and a story mode to complete to unlock new \
+karts and tracks. It features Grand Prix where the goal is to get the most points over \
+several races.
+
+The game also features additional game modes besides normal races : time trials, \
+follow-the-leader, soccer, capture-the-flag and two types of battle mode."
+HOMEPAGE="https://supertuxkart.net/
+	https://github.com/supertuxkart"
+COPYRIGHT="2020 SuperTuxKart Team"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/$portName/stk-code/releases/download/$portVersion/$portName-$portVersion-src.tar.xz"
+CHECKSUM_SHA256="052edf0afdbeb99583fe8676fb0ab80ecb6103fb88b7540f858d1b5fa1297d37"
+SOURCE_FILENAME="supertuxkart-$portVersion-src.tar.xz"
+SOURCE_DIR="SuperTuxKart-$portVersion-src"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	supertuxkart$secondaryArchSuffix = $portVersion
+	app:SuperTuxKart$secondaryArchSuffix
+	"
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfribidi$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcurl$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libglu$secondaryArchSuffix
+	devel:libglew$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libsdl2$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libbfd
+	devel:libfribidi
+	devel:libxcb
+	devel:libxfont
+	devel:libxrandr
+	devel:libxrender
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	rm -rf build
+	mkdir build
+	cd build
+
+	cmake -L .. \
```

Comment:
-L switch is only useful when doing a local build, this shouldn't go into haikuports (been there too) :)
```diff
- 	cmake -L .. \
+ 	cmake .. \
```